src/Sylius/Bundle/WebBundle/Resources/translations/messages.en.yml

```diff
@@ -47,6 +47,13 @@ sylius:
             tracking_number: Tracking number
         title: My account
     add_to_cart: Add to cart
+    association_type:
+        index_header: Association Types
```

Comment:
Same as higher, capital letters shouldn't be placed at the beginning of every word